conjureup/models/addon.py

```diff
@@ -32,9 +32,12 @@ def __init__(self, name):
         self.path = Path(app.config['spell-dir']) / 'addons' / name
         self.metadata = self._read('metadata.yaml')
         self.bundle = self._read('bundle.yaml')
-        self.steps = [StepModel.load(step_path)
+        self.friendly_name = self.metadata['friendly-name']
+        self.steps = [StepModel.load(step_path,
+                                     source=self.friendly_name)
```

Comment:
wat? why is the source arg the friendly name?

Reply:
It shows up in some error messages.  I started out using "spell" or the short name of the addon, which is fine for logging but I thought it made the error messages less helpful.  I'm fine changing it to the short name.